spec/validators/email_validator_spec.rb

```diff
@@ -0,0 +1,67 @@
+describe EmailValidator do
+  let(:model) { ExampleModel.new }
+
+  describe 'With nil email address' do
+    before do
+      model.email = nil
+      model.validate(:no_context)
+    end
+
+    it 'Returns invalid' do
+      expect(model.valid?(:no_context)).to be false
```

Comment:
the default is `:no_context` so why explicit?

Reply:
The default is `nil`, which causes `new_record?` (built-in for `ActiveRecord`) which isn't a method on the `ExampleModel`, but you can give it any context and it'll skip that so i explicitly named `:no_context`